resources/lang/en/patrols/mountainous/border/any.json

```
@@ -821,16 +904,19 @@
       "stat_skill": ["INSIGHTFUL,3", "CLEVER,2"],
       "relationships": [
         {
-        "cats_to": ["s_c"],
-         "cats_from": ["clan"],
-        "mutual": false,
-        "values": ["trust", "respect"],
-         "amount": 10
+            "cats_to": ["s_c"],
+            "cats_from": ["clan"],
+            "mutual": false,
+            "values": ["trust", "respect"],
+            "amount": 10,
+            "log": {
+                "cats_from": "to_cat brought valuable news of a wolf in the territory back to camp."
+            }
         }
       ]
     },
     {
-      "text": "s_c bolts back to camp, easily keeping ahead of the beast, {PRONOUN/s_c/subject} burst into camp, yowling a warning with plenty of time for c_n to prepare to drive the wolf away.",
+      "text": "s_c bolts back to camp, easily keeping ahead of the beast, {PRONOUN/s_c/subject} {VERB/s_c/burst/bursts} into camp, yowling a warning with plenty of time for c_n to prepare to drive the wolf away.",
```

```suggestion
      "text": "s_c bolts back to camp, easily keeping ahead of the beast. {PRONOUN/s_c/subject/CAP} {VERB/s_c/burst/bursts} into camp, yowling a warning with plenty of time for c_n to prepare to drive the wolf away.",
```

little too run-on as one sentence